frontend/src/libs/components/error-message/error-message.tsx

```diff
@@ -0,0 +1,20 @@
+import { getValidClassNames } from '~/libs/helpers/helpers.js';
+
+import styles from './styles.module.scss';
+
+type Properties = {
+  error?: string;
```

Comment:
Or `null` - cause actually we'd always pass this prop as a value from `errors` obj
```suggestion
  error: string | undefined;
```